raiden/transfer/mediated_transfer/events.py

```diff
@@ -32,6 +32,40 @@ def refund_from_sendmediated(send_lockedtransfer_event):
     )
 
 
+class SendLockExpired(SendMessageEvent):
+    def __init__(
+            self,
+            recipient: typing.Address,
+            message_identifier: typing.MessageID,
+            balance_proof: BalanceProofUnsignedState,
+            secrethash: typing.SecretHash,
+    ):
+        super().__init__(recipient, balance_proof.channel_identifier, message_identifier)
+
+        self.balance_proof = balance_proof
+        self.secrethash = secrethash
+
+    def __repr__(self):
+        return '<SendLockExpired msgid:{} balance_proof:{} secrethash:{} recipient:{}>'.format(
+            self.message_identifier,
+            self.balance_proof,
+            self.secrethash,
+            pex(self.recipient),
+        )
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, SendLockedTransfer) and
```

Comment:
Hehe ... copy pasta error here. We really should have unit tests for all those `__eq__` functions. Seen errors here more than 10 times. `SendLockedTransfer` should be `SendLockExpired`